src/nimblepkg/nimblesat.nim

```diff
@@ -488,10 +488,12 @@ proc getPackageMinimalVersionsFromRepo*(repoDir, pkgName: string, downloadMethod
   let taggedVersions = getTaggedVersions(repoDir, options)
   if taggedVersions.isSome:
     return taggedVersions.get.versions
-  gitFetchTags(repoDir, downloadMethod)       
+  gitFetchTags(repoDir, downloadMethod)    
+  var lastMinimalInfo = none(PackageMinimalInfo)   
```

Comment:
Doesn't look like it needs an option here.

Reply:
It can fail parsing the `.nimble` file